google-cloud-platform/index.md

```diff
@@ -9,7 +9,7 @@ has-permalinks: false
 # {{ page.title }} <img src="images/GoogleCloudPlatform-logo.png" alt="logo for Google Cloud Platform">
 
 **Note**: There are many ways to host a Dart server. You might also be
-interested in [Heroku][] and [Dartvoid][].
+interested in [Heroku][] and [Sourcevoid](https://www.sourcevoid.com/).
```

Comment:
When I go to https://raw.githubusercontent.com/Scorpiion/server-1/e6c085274d7f9cda3bf482ca769d0698e106fb5e/google-cloud-platform/index.md, the very last line is `[Dartvoid]: http://www.dartvoid.com/`.